### DISCUSSION TITLE
Process for getting a build into Maven

### DISCUSSION THREAD

**Discussion by arnavdugar-stripe** [Netflix/mantis#683] [General]
Hello, what is the process for making the build for [v3.0.20-rc.1](https://github.com/Netflix/mantis/releases/tag/v3.0.20-rc.1) available in Maven (e.g. [here](https://central.sonatype.com/artifact/io.mantisrx/mantis-common/3.0.19/versions))? Thanks!

cc @Andyz26 who was helping me with https://github.com/Netflix/mantis/pull/674.

**Comment by Andyz26** [comment]
I think that is the official release on maven. Does the RC build work as expected? If so I can submit an official release there.

**Reply by Andyz26** [reply]
for RC build it should be available from https://netflixoss.jfrog.io/ui/native/maven-oss-candidates/io/mantisrx/
